--- aboot-1.0_pre20040408.orig/zip/gzip.h	2003-11-07 16:03:36.000000000 -0800
+++ aboot-1.0_pre20040408/zip/gzip.h	2021-04-23 23:06:21.149598259 -0700
@@ -31,8 +31,8 @@
 #define WSIZE		 0x8000	/* window size--must be a power of two, and */
 				/*  at least 32K for zip's deflate method */
 
-unsigned char *inbuf;	/* input buffer */
-unsigned char *window;	/* sliding window and suffix table (unlzw) */
+extern unsigned char *inbuf;	/* input buffer */
+extern unsigned char *window;	/* sliding window and suffix table (unlzw) */
 
 extern unsigned insize; /* valid bytes in inbuf */
 extern unsigned inptr;  /* index of next byte to be processed in inbuf */
--- aboot-1.0_pre20040408.orig/include/bootfs.h	2021-04-24 02:06:04.241893045 -0700
+++ aboot-1.0_pre20040408/include/bootfs.h	2021-04-24 02:07:00.051158902 -0700
@@ -4,7 +4,7 @@
 #include <linux/types.h>
 #include <asm/stat.h>
 
-struct bootfs {
+extern struct bootfs {
 	int	fs_type;
 	int	blocksize;
 	
